Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "coreos-stream-metadata"
 version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
+rust-version = "1.60.0"
```

Comment:
Note that if we have any ambitions of using this crate in RHEL 8, e.g. in coreos-installer, this currently needs to be 1.58.0.

Reply:
Hmm, really?  http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/cargo-1.62.1-1.module_el8.7.0+1211+277be900.x86_64.rpm exists.

Reply:
That said, it's easy enough to just do https://github.com/coreos/stream-metadata-rust/pull/37 for now for consistency.

Reply:
Sorry, RHEL 8 CoreOS.  The OCP 4.12 buildroot still has 1.58.1.  I don't know if there are plans to bump Rust for 4.13, but usually we've stuck with what's currently available to us.

Reply:
I think we just need to ask to have it bumped.  I suspect going forward we could probably ensure we inherit latest RHEL8 - the problem AIUI is today it's a manual bump process.  But we just need to ask.

Reply:
I looked into it a bit more.  It looks like RHEL 8.6 and 9.0 still have Rust 1.58, though 1.62 is available in the 8.7 and 9.1 betas.